bigtable-dataflow-parent/bigtable-beam-import/README.md

```diff
@@ -168,15 +169,16 @@ Please pay attention to the Cluster CPU usage and adjust the number of Dataflow
     ```
 1. Run the import.
     ```
-    java -jar bigtable-beam-import-2.0.0-alpha1.jar import \
```

Comment:
isn't the latest version 2.0.1?

Reply:
We haven't released 2.0.1 yet I think? Currently we're on 2.0.1-SNAPSHOT. I don't think we're updating the version here with every release? But since we already released 2.0.0 I thought maybe we can get rid of the alpha version :)

Reply:
Thank you for fixing these - to have these update automatically, you can add the following before and after the version number. It should then be updated with each release:
```
[//]: # ({x-version-update-start:bigtable-client-parent:released})
<code goes here>
[//]: # ({x-version-update-end})
```